src/components/Form/index.js

```diff
@@ -48,19 +48,11 @@ export default class Form extends Component {
       }
 
       findNodesinDOM(props.children, ...formComponentNames)
-      .forEach(node => {
+      .forEach((node) => {
          this.state.data[node.props.name] = {
-<<<<<<< Updated upstream
-            value: node.props.value,
-            meta: {
-               error: null,
-               touched: false
-            }
-=======
             value: node.props.value || '',
```

Comment:
In one of my [previous commits](https://github.com/mother/oio/pull/15/commits/3d55182efd6343b7969f3ad99730e490387a1166), I removed blank strings set as default values, because this assumption is incorrect in the case of several kinds of form elements, including `Switch`, `CheckboxGroup`, and `RadioGroup`.